### DISCUSSION TITLE
Revision de todo el codigo

### DISCUSSION THREAD

**Discussion by MacaMorales** [CodeSystem2022/HW-ProyectoFinalPython#8] [General]
Estuve revisando el código cargado hasta el momento, y pude detectar que:

-  Se han empleado diferentes variables 
- Designación de clases con diferentes nombres
- Superposición de métodos

¿Cómo lo resolvemos?

**Comment by LucasEmir** [comment]
Buenas noches equipo.
Deberíamos separar las entidades y los servicios para dar más legibilidad al código.

**Comment by Valenttinna** [comment]
Hola! les parece que editemos las partes de codigo que le corresponde a cada uno nuevamente? Unificando los criterios.

**Comment by Marcos7gomez** [comment]
Buenas, si hay muchas cosas que cambiar podríamos empezarlo de cero y nos ponemos bien de acuerdo como lo vamos a ir codeando.

**Reply by MacaMorales** [reply]
Me parece que seria lo mejor. Aunque eso nos lleve más tiempo.

**Comment by florsha35** [comment]
ok, perfecto. Manos a la obra, que estamos sobre la fecha de entrega. Respetando el orden, modifiquen, controlen y suban el código.

**Comment by FranciscoTejada** [comment]
Perfecto, ahí empiezo a modificar mi parte del código y lo subo.
